ui/app/selectors/selectors.js

```diff
@@ -475,3 +481,11 @@ export function getIsSwapsChain(state) {
   const chainId = getCurrentChainId(state);
   return ALLOWED_SWAPS_CHAIN_IDS[chainId];
 }
+
+export function getPrimaryTokenImage(state) {
```

Comment:
Nit: The thing we're getting an image for here is referred to as the "native currency" throughout the extension. It is certainly not a token.

```suggestion
export function getNativeCurrencyImage(state) {
```